.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,6 +21,12 @@
 * [ ] TypeDoc updated (if applicable).
 * [ ] (Optional) Update documentation for new features.
 
+#### For React Hooks related changes:
```

Comment:
We could probably collapse this into the overall checklist and just have one list for any PR on the repo - keeps things succinct and leaves it unambiguous that the first checklist still applies in the case of React.

I think the live demo app doesn't need to be mentioned explicitly - this checklist also serves as a guide for external contributors and we won't necessarily expect them to go and make changes on another repo. Arguably once hooks are released, that would be our responsibility - as it would require a release to be created here, before the package is updated outside the repo.

Reply:
I have made the changes :)